src/apps/mailing/services.py

```diff
@@ -5,8 +5,26 @@
 from config import settings
 
 
+class TestMail:
+    """
+    Mailing class for tests to mock and check emails
+    """
+
+    mails: list[MessageSchema] = []
+
+    def __init__(self, connection):
+        self.connection = connection
+
+    async def send_message(self, message: MessageSchema):
+        self.mails.insert(0, message)
+
+    @classmethod
+    def clear_mails(cls):
+        cls.mails = []
```

Comment:
Why should we test this?
All external connections could be mocked.
But leave it for free if you need :)

Reply:
Anyway I think that `clear_mails` should be run automatically as far as it is a mandatory step for each test that triggers sending via email